### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by parano**
## What does this PR address?

Fixes #(issue)

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, `pre-commit run -a` script has passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,161 +1,173 @@
...
+* Support [all major ML frameworks](https://docs.bentoml.com/en/latest/frameworks/index.html), model formats, and model runtime.
+* Run and debug your BentoML apps locally on Mac, Windows, or Linux.
 
-✨ Looking deploy your ML service quickly? Checkout [BentoML Cloud](https://l.bentoml.com/bento-cloud)
-for the easiest and fastest way to deploy your bento.
+### 🍭 Simplify modern AI application architecture
 
-## Getting Started
+* Python-first design, effortlessly scale complex AI workloads.
+* Enable GPU inference with [a one-line config change](https://docs.bentoml.com/en/latest/guides/gpu.html), everything else auto-configured for you.
```
Comment (aarnphm) [inline README.md:26]:
```suggestion
* **Automatic resource allocation**: Enable GPU inference with [a one-line config change](https://docs.bentoml.com/en/latest/guides/gpu.html)
```

File: README.md [hunk 2]
```diff
@@ -1,161 +1,173 @@
...
+* Import from any model hub or bring your own models built with frameworks like PyTorch, TensorFlow, Keras, Scikit-Learn, and XGBoost.
+* Support [all major ML frameworks](https://docs.bentoml.com/en/latest/frameworks/index.html), model formats, and model runtime.
+* Run and debug your BentoML apps locally on Mac, Windows, or Linux.
 
-✨ Looking deploy your ML service quickly? Checkout [BentoML Cloud](https://l.bentoml.com/bento-cloud)
-for the easiest and fastest way to deploy your bento.
+### 🍭 Simplify modern AI application architecture
 
-## Getting Started
+* Python-first design, effortlessly scale complex AI workloads.
```
Comment (aarnphm) [inline README.md:25]:
```suggestion
* **Python-first**: Effortlessly scale complex AI workloads. With Python.
```

File: README.md [hunk 3]
```diff
@@ -1,161 +1,173 @@
...
+* Enable GPU inference with [a one-line config change](https://docs.bentoml.com/en/latest/guides/gpu.html), everything else auto-configured for you.
+* [Compose multiple models](https://docs.bentoml.com/en/latest/guides/graph.html) to run concurrently or sequentially [over multiple GPUs or Nodes](https://docs.bentoml.com/en/latest/guides/scheduling.html).
```
Comment (aarnphm) [inline README.md:27]:
```suggestion
* **Extensibility**: [Compose multiple models](https://docs.bentoml.com/en/latest/guides/graph.html) to run concurrently/sequentially [over multiple GPUs or Nodes](https://docs.bentoml.com/en/latest/guides/scheduling.html).
```

File: README.md [hunk 4]
```diff
@@ -1,161 +1,174 @@
...
 
-🏹 **_Scalable_** with powerful performance optimizations
+* Installation: `pip install bentoml`
+* Full Documentation: [docs.bentoml.com](https://docs.bentoml.com/en/latest/)
+* Tutorial: [Intro to BentoML](https://docs.bentoml.com/en/latest/tutorial.html)
 
-- [Adaptive batching](https://docs.bentoml.org/en/latest/guides/batching.html) dynamically groups inference requests on server-side optimal performance
-- [Runner](https://docs.bentoml.org/en/latest/concepts/runner.html) abstraction scales model inference separately from your custom code
-- [Maximize your GPU](https://docs.bentoml.org/en/latest/guides/gpu.html) and multi-core CPU utilization with automatic provisioning
+### 🛠️ What you can build with BentoML
```
Comment (ssheng) [inline README.md:51]: Should we just link Gallery so we have one place to keep all deployment. We should move these descriptions to to gallery, too. I made a feature request to #design for adding gallery descriptions.

**Pull request merged**